### DISCUSSION TITLE
Terrains other formats

### DISCUSSION THREAD

**Discussion by radoslavirha** [visgl/loaders.gl#2193] [Ideas]
Hi,

is it possible to support other terrain elevation formats? Like `lerc` from ESRI? Now I have to proxy requests through API, decode it and return `png`.

Thanks

**Comment by kylebarron** [comment]
`lerc` isn't a file format, it's a compression algorithm. Are you referring to a geotiff with lerc compression? You can use geotiff.js to load it: https://github.com/geotiffjs/geotiff.js

**Reply by radoslavirha** [reply]
Don't know which format it is, it returns an array of pixels where each pixel is altitude. I'll take a look at geotiff! I did not know it works with lerc. Maybe it's just geotiff. Will take a look.

Thanks
